### DISCUSSION TITLE
set exporter.fhir.use_us_core_ig = false when we want to generate US Core ?

### DISCUSSION THREAD

**Discussion by kalyanmysore** [synthetichealth/synthea#1538] [Q&A] [answered]
In the github documentation page: [https://github.com/synthetichealth/synthea/wiki/HL7-FHIR](https://github.com/synthetichealth/synthea/wiki/HL7-FHIR)
It says to set exporter.fhir.use_us_core_ig = false
And then set exporter.fhir.us_core_version = 5.0.1

Are we sure we want to set exporter.fhir.use_us_core_ig = false when we want to generate US Core ?

**Comment by jawalonoski** [comment]
That isn't an instruction, it is an example.

**Reply by jawalonoski** [reply, marked as answer]
Yes, if you want US Core, you'd set that to `true`
